test/server/core/card/CardHighlighting.spec.ts

```diff
@@ -0,0 +1,67 @@
+describe('Card highlighting', function() {
+    integration(function(contextRef) {
+        describe('Card highlighting', function() {
+            beforeEach(function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        resources: ['armed-to-the-teeth',
+                            'collections-starhopper',
+                            'covert-strength',
+                            'chewbacca#pykesbane',
+                            'battlefield-marine',
+                            'moment-of-peace',
```

Comment:
protip: can use 'atst' for easy copy-paste

Reply:
will keep that in mind for next time!